packages/strapi-plugin-upload/admin/src/components/SortPicker/index.js

```diff
@@ -1,14 +1,68 @@
-import styled from 'styled-components';
-
-const SortPicker = styled.div`
-  height: 32px;
-  padding: 0 10px;
-  background: #ffffff;
-  line-height: 30px;
-  border: 1px solid #e3e9f3;
-  border-radius: 2px;
-  font-size: 13px;
-  font-weight: 500;
-`;
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import getTrad from '../../utils/getTrad';
+
+import Wrapper from './Wrapper';
+import SortButton from './SortButton';
+import SortList from './SortList';
+import SortListItem from './SortListItem';
+
+const SortPicker = ({ onChange, value }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const orders = {
+    created_at_asc: 'created_at:ASC',
+    created_at_desc: 'created_at:DESC',
+    name_asc: 'name:ASC',
+    name_desc: 'name:DESC',
+    updated_at_asc: 'updated_at:ASC',
+    updated_at_desc: 'updated_at:DESC',
+  };
+
+  const handleChange = value => {
+    onChange({ target: { value } });
```

Comment:
Here you can add a `name` to the `target` so you are in the same situation as a normal form.